### ISSUE TITLE
[BITV] 12.1.2/0.1 - The web application provides all users a page (https://try.nextcloud.com/ltd/a11y/index.php/settings/help) with the product documentation.

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/documentation#9748]
This documentation does not meet all the criteria of EN 301 549 Chapter 9 for websites. (1)

- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cfb0a8a23a-v0-n1

</details>

**Comment by skjnldsv** [comment]
@AndyScherzinger I don't get that one 🤔

**Comment by AndyScherzinger** [comment]
Me neither, so let's close it. The report document also had no further details. "The internet" says:

Chapter 9 ensures that the following aspects of a website meet the necessary accessibility standards:

* **Text Alternatives**: Providing alternatives for non-text content.
* **Adaptability**: Creating content that can be presented in different ways (e.g., simpler layout) without losing information.
* **Distinguishability**: Making it easier for users to see and hear content, including separating foreground from background.
* **Keyboard Accessibility**: Ensuring all functionality is available via a keyboard.
* **Navigability**: Providing ways to help users navigate, find content, and determine where they are.
* **Input Modalities**: Making it easier to use functions via inputs other than keyboards.
* **Readability**: Making text content readable and understandable.
* **Predictability**: Making web pages appear and operate in predictable ways.
* **Input Assistance**: Helping users avoid and correct mistakes.
* **Compatibility**: Maximizing compatibility with current and future user agents, including assistive technologies

So I'd say fine to close
